src/hooks/usePage.ts

```diff
@@ -0,0 +1,16 @@
+import { useMemo } from 'react'
+
+/**
+ * This hook returns the correct page (defined as a slice of a passed in array) given a dataset, the items per page
+ * that are to be displayed, the wanted page number (NON zero based) and the number of
```

Comment:
By "NON zero based" do you mean not zero indexed?

Reply:
Oh, yeah, changing that.